index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Microverse Portfolio</title>
+</head>
+<body>
+    <section class="header">
+        <nav>
+            <p>My Logo</p>
+            <img src="./assets/menu.png" alt="logo">
+        </nav>
+        <div class="background">
+            <div class="intro">
+                <div class="text">
+                    <h3>I'm Favour. <br> Glad to see you!</h3>
+                </div>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                
+                <div class="socials">
+                    <h5>LET'S CONNECT</h5>
+                    <div class="social-icon">
+                        <a href="#"> <img src="./assets/twitter.svg" alt="twitter"> </a>
+                        <a href="#"> <img src="./assets/Linkedin icon.svg" alt="Linkedin"> </a>
+                        <a href="#"> <img src="./assets/medium.svg" alt="medium"> </a>
+                        <a href="#"> <img src="./assets/github.svg" alt="github"> </a>
+                        <a href="#"> <img src="./assets/angellist.svg" alt="angellist"> </a>
+                    </div>
```

Comment:
- [ ] Please use a list here instead of a `div` as this is a list of social media links and can be represented using `ul` and `li`.